src/hooks/use-audit-logs.ts

```
@@ -1,267 +1,311 @@
+    case 'maintenance_plan_update':
+      return getDetailString(details, 'plan_name')
+        ? `Kế hoạch: ${getDetailString(details, 'plan_name')}`
+        : ''
+    case 'transfer_create':
+    case 'transfer_update':
+      return getDetailString(details, 'equipment_name')
+        ? `TB: ${getDetailString(details, 'equipment_name')}`
+        : ''
+    default:
+      // Try to extract meaningful info from details
+      const description = getDetailString(details, 'description')
```

Biome (2.4.10)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)